mkdocs.yml

```diff
@@ -103,6 +103,10 @@ nav:
       - Chat History: "concepts/chat_history.md"
       - Tools (Function Calling): "concepts/tools_(function_calling).md"
       - Extracting structured information using LLMs: "concepts/extracting_structured_information_using_llms.md"
+      - Evaluations: 
```

Comment:
hmm, I feel like this should just be it's own section in concepts? We already reference you should read it at the top, and I think nesting it will make it harder to find.